packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -729,26 +729,31 @@ export class TreeWidget extends ReactWidget implements StatefulWidget {
      * @param node the tree node.
      * @param icon the icon.
      */
-    protected decorateIcon(node: TreeNode, icon: React.ReactNode | null): React.ReactNode {
-        // eslint-disable-next-line no-null/no-null
-        if (icon === null) {
-            // eslint-disable-next-line no-null/no-null
-            return null;
+    protected decorateIcon(node: TreeNode, icon: React.ReactNode): React.ReactNode {
+        // icon can be null or undefined
```

Comment:
I don't think this comment is necessary - the `!icon` check makes it clear you're checking for all falsey values, and no one who isn't checking this diff will know that the code _used_ to assume only `null` was a possibility. :-)

Reply:
I was surprised and I asked Omar to add that comment... `ReactNode` is a weird type, but I assume you are right, just need to `Go To Type Definition` and we can see that it includes null.

Reply:
I will remove the comment thanks !